README.md

```diff
@@ -1,54 +1,239 @@
+My excitement level: <|{excitement}|text|>
+"""
+excitement=100
+
+Gui(page=my_page).run()
+```
+*RUN*🏃🏽‍♀️  
+<div align="center">🎊 TADA! 🎊</div>  
+<br>
+<div align="center"><img src="readme_img/readme_gui_app.gif" width=600 height=400 alt="GUI demo"></img></div>  
+
+<br>
+<br>
+
+### <div align="center">*GUImme more*</div>
+*<div align="center">Check out our [Getting Started](https://docs.taipy.io/en/latest/getting_started/getting-started-gui/) and [Documentation](https://docs.taipy.io/en/latest/manuals/gui/)</div>*
+
+<br>
+<br>
+
+## EN-CORE?
+
+#### <div align="center">Let's create a back-end execution, also called scenario using Taipy Core. Our scenario will filter movie data based on the genre you choose. This scenario will be submitted (i.e., executed) each time the genre selection changes and output the seven most popular movies of that genre. </div>  
+<br>
+
+*Here is our filter function: it's a standard Python function that will constitute our unique task in the scenario*
+```python
+def filtering_genre(initial_dataset: pd.DataFrame, selected_genre):
+    filtered_dataset = initial_dataset[initial_dataset['genres'].str.contains(selected_genre)]
+    filtered_data = filtered_dataset.nlargest(7, 'Popularity %')
+    return filtered_data
+```
+
+*This is the execution graph of the scenario we will be implementing*
+
+<div align="center"><img src="readme_img/readme_exec_g.png" alt="Taipy Logo"  width="400" height="300"/></div> 
+
+
+### Taipy Studio - The easy peasy way
+*You can use the Taipy Studio extension in VSCode to configure your pipeline easily* 
+
+<div align="center"><img src="readme_img/readme_studio.gif" width=600 height=400 alt="GUI demo"></img></div> 
+
+*Your configuration is automatically saved as a toml file* 
+
+<br>
+<br>
+
+### <div align="center">*Want to be Studio-us?*</div>
+*<div align="center">Check out our [Getting Started](https://docs.taipy.io/en/latest/getting_started/getting-started-core/) and [Documentation](https://docs.taipy.io/en/latest/manuals/studio/) </div>*
+
+<br>
+<br>
+<br>
+<br>
 
-## Contributing
+### Taipy CORE - a walk on the code side
+<div align="left">If you prefer coding your configurations instead of using Taipy Studio, Taipy's got you 🫵 </div>   
+
+*<div align="left">Check out the movie genre demo scenario creation with this [Demo](https://www.taipy.io/project/movie-genre-selector/) </div>*
+
+<br>
+<br>
+<br>
+
+### <div align="center">*Gimme Core*</div>
+*<div align="center">Check out our [Getting Started](https://docs.taipy.io/en/latest/getting_started/getting-started-core/) and [Documentation](https://docs.taipy.io/en/latest/manuals/core/) </div>*
+
+<br>
+<br>
+<br>
+
+
+## Front-end ➕ Back-end
+*Now, let's load this configuration and add a GUI on top for a 🎉FULL APPLICATION🎉*
+```python
+import taipy as tp
+import pandas as pd
+from taipy import Config, Scope, Gui
+
+# TAIPY Core
+
+# Filtering function - task
+def filtering_genre(initial_dataset: pd.DataFrame, selected_genre):
```

Comment:
Minor remark: 
I would rename it `filter_genre`:
Usually, we prefer to use an infinitive verbal form for naming functions: `filter_genre`. It represents more actions to execute. Nominal forms are more for variables representing more a result, a static value, or a state.